docs/api/api-locally.md

```diff
@@ -0,0 +1,31 @@
+# Running the API Locally
+
+## Testing Your Code Locally
+
+In order to test our code locally, we have a few options. We can use in-built web-request commands that are featured within your Operating System's terminal or we can use a REST API Caller program like [Postman](https://www.postman.com/) or [Insomnia](https://insomnia.rest/). You can access them and download one by hitting the respective links.
+
+To run your API locally, we must now get our terminal to be in the same folder as our root folder for our API. If you opened your terminal via VS Code, you can skip this next step and get straight to running some commands!
+
+## Getting your Terminal to the right directory
```

Comment:
Might be easier to just mention to continue using the vs code terminal since it is already in the right location?